settings.py

```diff
@@ -64,7 +64,8 @@
 else:
   logging.error('Unexpected app ID %r, please configure settings.py.', APP_ID)
 
-SECRET_KEY = os.environ['DJANGO_SECRET']
+# SECRET_KEY = os.environ['DJANGO_SECRET']
+SECRET_KEY = "test secret"
```

Comment:
Hard Coding secrets in the code itself doesn't make sense. Also this is not a good practise.

Reply:
I forgot not to add this change. Thank you @whoanuragverma. I will change it